### PR TITLE
Updating documentation for clarity

### LINKED ISSUE #51: Update Contributing document to add backend state and CLI commands
### Overview

The contributing guide includes a step to create a backend state that links to an issue. It's better to include these instructions directly within the contributing doc. It would also be helpful to include CLI commands for anyone who would prefer not to use the AWS UI

### Action Items

- [ ] Update the contributing guide to include steps to add the backend state
- [ ] Add CLI commands to create resources

### Resources/Instructions

[CONTRIBUTING.md](https://github.com/hackforla/devops-security/blob/main/CONTRIBUTING.md#creating-backend-state)

### PR DESCRIPTION
**Pull request by chelseybeck**
- Resolves #51 

Changes:
Updated Contributing doc and pre-work template to clarify instructions and incorporate referenced links

Also updated README w/ Terraform Docs to reflect recent addition of resources and added that as a step

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/pre-work-template-devops-security.md [hunk 1]
```diff
@@ -63,12 +65,25 @@ As a new member on the HfLA devops-security team, fill in the following fields a
...
-      - [ ] ** If you have cost concerns, Run a Terraform Destroy to take down all of the resources you created (don't worry, you can recreate them just as quickly). **
+- [ ] Then execute the command `terraform apply` this will create all of the resources that are currently managed by Devops Security. All of the resources created here incur zero cost except for the Dynamo DB installation, which should remain in the free tier. **
+  - [ ] **If you have cost concerns, Run a Terraform Destroy to take down all of the resources you created (don't worry, you can recreate them just as quickly). If you create resources outside of what's described in this issue, you may incur charges.**
+- [ ] Update the README using Terraform Docs to document changes
+  - [ ] ```terraform-docs -c .terraform.docs.yml .```
 - [ ] Once you have tested your changes, stage them in git with 
     - [ ] `git status` command.
     - [ ] then `git add path/to/file` (you can copy from above output for the file path).
-- [ ] Commit the changes by executing `git commit -m "briefly describing the changes"`.
+- [ ] Commit the changes by executing `git commit -m "briefl description of changes"`.
```
Comment (paulespinosa) [inline .github/ISSUE_TEMPLATE/pre-work-template-devops-security.md:88]: spelling: "brief"

**Review by paulespinosa — APPROVED** [review]
It looks good @chelseybeck! On Slack you asked about repeating some steps on both CONTRIBUTING and the issue template. I don't have the foresight to know how much of a problem this would cause if they ever get out of sync. However, I did like being able to refer to only the issue template for those simple one line commands.

This review does not include scrutinizing or attempting the steps.

I approve the changes leaving the requested/suggested changes to your discretion. Thank you.

File: .github/ISSUE_TEMPLATE/pre-work-template-devops-security.md (hunk 1 above)
Comment (sudhara) [inline .github/ISSUE_TEMPLATE/pre-work-template-devops-security.md:88]:
```suggestion
- [ ] Commit the changes by executing `git commit -m "brief description of changes"`.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -158,15 +196,22 @@ To facilitate AWS IAM changes using Terraform, it's essential to establish backe
 
 Atfer creating a backend state, create a ```backend.tfvars``` file in the ```terraform``` directory. It should have content of this format:
 
-```
-bucket         = "{developer_specific}-hfla-ops-terraform-state"
+```bash
```
Comment (sudhara) [inline CONTRIBUTING.md:199]:
```suggestion
```terraform
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -158,15 +196,22 @@ To facilitate AWS IAM changes using Terraform, it's essential to establish backe
 
 Atfer creating a backend state, create a ```backend.tfvars``` file in the ```terraform``` directory. It should have content of this format:
```
Comment (sudhara) [inline CONTRIBUTING.md:197]:
```suggestion
After creating a backend state, create a `backend.tfvars` file in the `terraform` directory. It should have content of this format:
```

**Review by sudhara — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/pre-work-template-devops-security.md [hunk 4]
```diff
@@ -9,7 +9,7 @@ assignees: ''
 ---
 
 ### Prerequisite
-We are looking forward to having you on our team. Please ensure you have prior experience with the HfLA website team before contributing to our repository.
+We are looking forward to having you on our team. Please ensure you have prior experience with the HfLA website team before contributing to our repository
```
Comment (sudhara) [inline .github/ISSUE_TEMPLATE/pre-work-template-devops-security.md:12]:
```suggestion
We are looking forward to having you on our team. Please ensure you have prior experience with the HfLA website team before contributing to our repository.
```

File: .github/ISSUE_TEMPLATE/pre-work-template-devops-security.md [hunk 5]
```diff
@@ -35,7 +35,9 @@ As a new member on the HfLA devops-security team, fill in the following fields a
     - [ ] [Generating user access keys](https://github.com/hackforla/devops-security/blob/main/CONTRIBUTING.md#generating-access-keys-for-aws-cli)
 - [ ] Complete the instructions in [AWS Documentation](https://docs.aws.amazon.com/cli/v1/userguide/cli-chap-install.html) and choose your operating system to install AWS CLI. 
 - [ ] Complete the instruction in [AWS Documentation](https://docs.aws.amazon.com/cli/latest/userguide/cli-authentication-short-term.html) to setup the AWS CLI.
-- [ ] Read follow the instructions in [Creating a backend state](https://github.com/hackforla/devops-security/blob/main/CONTRIBUTING.md#creating-backend-state).
+- [ ] Follow the instructions in [Creating a backend state](https://github.com/hackforla/devops-security/blob/main/CONTRIBUTING.md#creating-backend-state).
```
Comment (sudhara) [inline .github/ISSUE_TEMPLATE/pre-work-template-devops-security.md:38]:
```suggestion
- [ ] Follow the instructions in [Creating a backend state](https://github.com/hackforla/devops-security/blob/main/CONTRIBUTING.md#creating-backend-state) to create the S3 bucket and DynamoDB table.
```

**Review by sudhara — APPROVED** [review]
Resolved the outstanding comments during the CoP DevOps meeting.

**Pull request merged**